### ISSUE TITLE
Back up codes online/offline

### ISSUE THREAD

**Issue by Toucan-Sam** [arturogutierrez/Openticator#81]
TL;DR: This feature already has a holding place in the app and is "Coming Soon".

Pretty much the singular feature that Authy has over any other 2FA app is the ability to back up your codes in case you lose your device.

I know at least Google can use an SMS as an alternative to an OTP in the event of losing access to your device. Adding a feature to (securely) back up your codes to Google Drive, DropBox, NextCloud, etc, would be fantastic.

Even an encrypted export option so you can store the backup somewhere yourself in the event of losing your device.

This is a long-term feature request and just opening it as a bookmark as an "existing issue". :smile:
